core/config/job-distributor.go

```diff
@@ -0,0 +1,5 @@
+package config
+
+type JobDistributor interface {
+	NopFriendlyName() string
```

Comment:
I agree with @jmank88 's suggestion. Let's call it "NopDisplayName"? Or maybe even "NodeDisplayName"? Ideally, we want this string to contain both the name of the NOP and the Node, right?

Reply:
Correct -- it should be the fully qualified name that we can use when talking to NOPs that will allow them to identify the node without ambiguity :)